client/components/CheckoutTool/CheckoutTool.js

```diff
@@ -4,9 +4,9 @@ class CheckoutTool extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      pricePerNight: '',
-      cleaningFee: '',
-      serviceFee: '',
+      pricePerNight: this.props.pricePerNight,
```

Comment:
should you store state in 2 components? im not sure if you need to store  state in CheckoutTool component. you would have to update state in two places based on user input.

Reply:
will edit it out to receive props only and use some sort of update component to keep track of prop change